apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -1,11 +1,49 @@
-/*global BridgeServiceMixin */
+/*global bridge,
+         BridgeServiceMixin,
+         BroadcastChannel,
+         Drafts,
+         streamClient,
+         Thread
+*/
 'use strict';
 
-if (!('BridgeServiceMixin' in self)) {
-  importScripts('/services/js/bridge_service_mixin.js');
-}
+/**
+ * Type description for the ConversationSummary object that is return from the
+ * Conversation service.
+ * @typedef {Object} ConversationSummary
+ * @property {number} id Unique identifier of the the conversation.
```

Comment:
mmm ok, I likely thought of the DOM id and what I would like to see for draft/conversation ids in my mind ;)

I'm mainly afraid of collisions if we don't have a 100%-safe way to generate different ids for drafts and conversations. Maybe when we'll have a gaia-based DB we can ensure this with numbers but nowadays I don't see how this is reliably possible. We're just lucky for now ;)